metrics/first-meaningful-paint/audit.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+var DevtoolsTimelineModel = require('devtools-timeline-model');
+
+module.exports = function(data) {
+  // FIXME(paulirish) save the trace to file in debug file for later analysis.
```

Comment:
we should make a real issue for tracking this. I think we're going to want an easy way to save artifacts in general (e.g. for graphing over time a la bigrig)